lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/AllocatedEvaluator.java

```diff
@@ -65,9 +65,17 @@
    *
    * @param type
    */
+  @Deprecated
```

Comment:
Actually, I don't believe this is true. AllocatedEvaluator is `@Public` and the way for user code to start a CLR evaluator from Java would be to use this method (or now, `setProcess`). This is illustrated in `org.apache.reef.examples.helloCLR`.